### ISSUE TITLE
Link editing tools are not keyboard accessible (esp. for screen readers)

### ISSUE THREAD

**Issue by jeyoshimi** [fraywing/textAngular#1533]
Reproducible by clicking on the link in the editor demo:
http://plnkr.co/edit/hz3x2f?p=preview
When you left click on the link, a panel pops up allowing you to edit, delete or open link in new window. This functionality is currently inaccessible to screen reader users (or users who rely on keyboard navigation to use the editor) as per WCAG 2.0 guideline [2.1.1 (Keyboard)](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/keyboard-operation-keyboard-operable.html).

With the cursor inside of the link text in the textarea, hitting TAB should go to the link inside the link-modify widget. TAB again should go to the pencil ("Edit link") button. TAB again should go to the circular ("Remove link") button. SHIFT-TAB should take you back through the elements in reverse order.

Note: The buttons have tabindex="-1" attribute, which means users can't focus on them. The panel also seems to be inserted before the textarea in the DOM, which is relevant to WCAG 2.0 Guidelines [1.3.2 (Meaningful Sequence)](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/content-structure-separation-sequence.html) and [2.4.3 (Focus Order)](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/navigation-mechanisms-focus-order.html).

Browser: Chrome

**Comment by stevenkissack** [comment]
I see this being low priority right now as the library does not have many maintainers, if you'd like to submit a pull request for this feature I am sure it will be welcomed.
